src/test/dir.proj

```diff
@@ -46,13 +46,13 @@
     <PropertyGroup>
       <RestoreSourceArg>@(RestoreTestSource->'--source %(Identity)', ' ')</RestoreSourceArg>
       <RestoreSourceArg>$(RestoreSourceArg) @(RestoreTestFallbackSource->'--source %(Identity)', ' ')</RestoreSourceArg>
-      <RestoreArgs>--disable-parallel --packages "$(PackagesDir.TrimEnd('/\'.ToCharArray()))" $(RestoreSourceArg)</RestoreArgs>
+      <RestoreArgs>--disable-parallel --packages "$(PackagesDir.TrimEnd('/').TrimEnd('\'))" $(RestoreSourceArg)</RestoreArgs>
       <RestoreArgs>$(RestoreArgs) /p:MNAVersion=$(ProductVersion) /p:TestTargetRid=$(TestTargetRid)</RestoreArgs>
       <RestorePrefix Condition="'$(OsEnvironment)' != 'Windows_NT'">DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1</RestorePrefix>
```

Comment:
I'm not sure. I think we wanted the flexibility to use different package sources from the default here, but ultimately it looks like we are kind of using the same options, more-or-less, plus some extras.